sales_tax_using_oop.rb

```diff
@@ -0,0 +1,10 @@
+$LOAD_PATH << '.'
```

Comment:
There is one more way to import file without using **$LOAD_PATH << '.'** try it.